src/views/evaluations/evaluations.js

```diff
@@ -0,0 +1,98 @@
+function EvaluationsPage (options = {}) {
+    const headings = [
+        'Nume','Tehnologie','Nivel'
+    ]
+    const rows = [
+        {
+            name: "Popescu Adrian",
+            technology: "Javascript",
+            level: "Mid 2"
+        },
+        {
+            name: "Dragan Roxana",
+            technology: "PHP",
+            level: "Mid 1"
+        },
+        {
+            name: "Florescu Mihai",
+            technology: "Javascript",
+            level: "Junior 3"
+        },
+        {
+            name: "Gheorghe Andrei",
+            technology: "Ruby",
+            level: "Senior 1"
+        }
+    ]
+    return `
+    ${NAV()}
+    ${EvaluationsTable({
+        items: rows,
+        itemHeadings: headings
+    })}
+    ${Footer()}
+    `;
+}
+
+function EvaluationsTableHeader(options) {
+    const headings = [];
+    options.headings.forEach(function(el) {
+        const generatedEl = `<th class="candidates_table_header">${el}</th>`;
+        headings.push(generatedEl);
+    });
+    const headingsEl = `
+    ${headings.join('')};`
+    return `
+    <tr>
+        ${headingsEl}
+    </tr>
+    `;
+}
+
+function EvaluationsTableRow (options = {}) {
+    return `
+    <tr class="rows">
+        <td>${options.name}</td>
+        <td>${options.technology}</td>
+        <td>${options.level}</td>
+        <td style="text-align: right">Detalii <button>+</button></td>
+    </tr>
+    `
+}
+
+function EvaluationsTableBody (options = {}) {
+    //const rowsElements = [];
```

Comment:
Commented code goes away as soon as alternative is confirmed to work properly, yes? :stuck_out_tongue:

Reply:
Right! :)